test/collection_test.js

```diff
@@ -2859,6 +2859,13 @@ describe("Collection", () => {
         .then(result => expect(result.data._status).eql("deleted"));
     });
 
+    it("should support deleteAll", () => {
+      articles.deleteAll();
+      articles
+        .list({ filters: { _status: "deleted" } }, { includeDeleted: true })
+        .then(result => expect(result.data._status).eql("deleted"));
```

Comment:
I don't believe this work :thinking: 
`list()` returns a list of records, I can't see how `result.data._status` could be equal to `deleted`

Reply:
why not, I'm calling `delete` within `deleteAll` which should set the `_status` to `"deleted"` so it should work :)

Reply:
But `result` here, in the `then()` takes the result of the `list()` method call, which is an array. This code reads the `data` attribute of the array, that's why I'm puzzled!

Reply:
I can trace that down, to see why this is working!